javascript/javascript1/week2/homework4/smartEase.js

```diff
@@ -0,0 +1,29 @@
+
+function getFullnameV1(firstName, surname){
+  let fullName = `${firstName} ${surname}`
+  return fullName;
+}
+
+const fullName1 = getFullnameV1('Mark', 'Corrigan');
+const fullName2 = getFullnameV1('Jeremy', 'Usbourne');
+console.log(fullName1);
+console.log(fullName2);
+
+function getFullnameV2(firstName, surname, useFormalName=true, gender) {
+    if((useFormalName === undefined || useFormalName === false) && (gender === undefined || gender === 'other')){
+        return getFullnameV1(firstName, surname)
+    }
+    else if(useFormalName === true){
```

Comment:
As a general rule of thumb, I'd suggest avoiding comparing with booleans as you can simple do `if (useFormalName)` and it behaves identically! 😄 Can you think of how we'd check for something being false without using a comparison? (Hint: you will need an operator!)

Reply:
How did I miss that !!!
I cleaned the code, introduced a fallback value for gender
Thank you so much 🙏🙏